DeveloperGame/DeveloperGame/DeveloperGameClass.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+
+public class DeveloperGameClass
```

Comment:
Don't put "Class" in the name of a class. This is true when creating almost any type

Reply:
I fixed this issue.